src/storage/index.ts

```diff
@@ -25,21 +27,143 @@ export const deleteConferenceSettings = () => {
 // Settings
 const SETTINGS_KEY = "SettingsKey";
 
+interface saveSettingFormat {
+  active: boolean;
+  encoded: string;
+  id: number;
+}
+
 export const saveSettings = (settings: AppSettings) => {
   const encoded = Buffer.from(JSON.stringify(settings), "utf-8").toString(
     "base64"
   );
 
-  localStorage.setItem(SETTINGS_KEY, encoded);
+  const str = localStorage.getItem(SETTINGS_KEY);
+  if (str) {
```

Comment:
const str = localStorage.getItem(SETTINGS_KEY);
const parsed = str ? JSON.parse(str) : [];
const newItem = { id: parsed.length + 1, encoded, active: parsed.length === 0 };

localStorage.setItem(SETTINGS_KEY, JSON.stringify([...parsed, newItem]));

Could you rethink to make it simpler like this?

Reply:
okay, on it